src/Router.ts

```diff
@@ -61,10 +61,7 @@ export type RouteNavigationFunction = (route: RoutedLocation, animation?: RouteA
  *   <omni-router-outlet></omni-router-outlet>
  * ```
  */
-export class Router {
-
-	/**  The singleton instance of the router. */
-	private static _instance: Router;
+export class RouterImpl {
```

Comment:
The effect of this is that the module named import will now look this:

```
import { RouterImpl } from '@capitec/omni-router';
```

For this change, I would have opted to:

1. Leave the class name as is.
OR
2. Not export the class (with any name), just have a the default export as `Router`.

Given the deprecation of `getInstance()`, I would have opted for option 1.

Reply:
oversight, the Impl should not be exported, the last line in the file is the named export